src/main/java/org/folio/InvoiceWorkflowDataHolderBuilder.java

```diff
@@ -143,13 +143,13 @@ public List<InvoiceWorkflowDataHolder> checkMultipleFiscalYears(List<InvoiceWork
       .filter(h -> h.getBudget() != null && h.getBudget().getFiscalYearId() != null)
       .collect(groupingBy(h -> h.getBudget().getFiscalYearId()));
     if (fiscalYearToHolders.size() > 1) {
+      logger.warn("More than one fiscal years found");
```

Comment:
adding name of method is highly recommended

Reply:
Except for log.error, because error stack trace already shows the location of error

Reply:
for me log.error is more appropriate, but it is up to you, log.warn also may be acceptable

Reply:
done